### DISCUSSION TITLE
How can I get the predicted fluxs of gaussian process('sklearn')

### DISCUSSION THREAD

**Discussion by Yzxastr** [phoebe-project/phoebe2#653] [Questions]
I have abb three kernels in gaussian process , but I don't know how to get the predicted fluxs after I use the following code 'b.run_compute(model='model_gps', overwrite=True)' . Could you tell me how to get the predicted fluxs of GPs . 
If possible, I would like to ask how to use phoebe get the light curves after subtracting the quasi-sinusoidal signals from GPs .

**Comment by kecnry** [comment]
The fluxes with and without GPs are included as parameters in the model.  The [gaussian processes tutorial](http://phoebe-project.org/docs/latest/examples/minimal_GPs) shows how to plot either of these, and the [computing forward models tutorial](http://phoebe-project.org/docs/latest/tutorials/compute) shows examples of accessing the arrays from the model parameters.
